File: VetAPI/Models/Alimentacao.cs
namespace VetAPI.Models
{
    public class Alimentacao
    {
        public int AlimentacaoId { get; set; }
        public DateTime DataHoraAlimentacao { get; set; }
        public string? TipoAlimento { get; set; }
        public decimal QuantidadeFornecida { get; set; }
        public string? FrequenciaRefeicoes { get; set; }
        public string? Observacoes { get; set; }

        // Relação com o animal (caso cada registro de alimentação seja associado a um animal específico)
        public Animal? Animal { get; set; }
    }
}

File: VetAPP/src/app/Alimentacao.ts
import { Animal } from './Animal';

export class Alimentacao{
    alimentacaoId: number = 0;
    dataAlimentacao: Date;
    horaAlimentacao: string = '';
    tipoAlimento: string = '';
    quantidadeFornecida: number = 0;
    frequenciaRefeicoes: string = '';
    observacoes: string = '';
    animal: Animal | undefined;

    constructor(alimentacaoId: number, dataAlimentacao: Date, horaAlimentacao: string) {
        this.alimentacaoId = alimentacaoId;
        this.dataAlimentacao = dataAlimentacao;
        this.horaAlimentacao = horaAlimentacao;
    }
}